qutebrowser/completion/completer.py

```diff
@@ -197,13 +197,19 @@ def schedule_completion_update(self):
         For performance reasons we don't want to block here, instead we do this
         in the background.
         """
-        if (self._cmd.cursorPosition() == self._last_cursor_pos and
+        cmd = self._cmd.text().split(' ', 1)
+        if (len(cmd) > 1 and
```

Comment:
Is this so the command completion is immediate? (might want to explain this in the docstring or so)